test/coverlet.core.performancetest/coverlet.core.performancetest.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(MSBuildThisFileDirectory)\..\..\build\$(Configuration)\coverlet.msbuild.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="xunit" Version="2.3.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\coverlet.testsubject\coverlet.testsubject.csproj" />
```

Comment:
I think we can move `BigClass.cs` directly into this project.

Reply:
I might have misunderstood something then, as I thought only the dependencies of the unit tests got instrumented, not the unit test DLL itself. Is that not the case? The purpose of having this test subject DLL was to get BigClass instrumented when running the performance test.

Reply:
My mistake @petli. Thanks for the clarification

Reply:
The class do need a comment block explaining why it is where it is, I'll fix that.